microsoft/onefuzz
# handle libfuzzer targets failing without dropping crash files

## Summary of the Pull Request

Handle libfuzzer targets exiting with an error without recording crash files

## PR Checklist
* [X] Applies to work item: #57 

## Info on Pull Request

* Moves `src/process.rs` from `onefuzz-supervisor` to `onefuzz`
* Keeps a copy of stderr in case the fuzzer crashes without dropping crash file
* Adds `onefuzz-agent debug libfuzzer_fuzz` debug subcommand to test libfuzzer targets locally

## Validation Steps Performed

* Deploy a regular libfuzzer task and note it detects crashes as appropriate 
* Deploy a target that does `raise(SIGKILL);` and note that we mark the task is failed
